prime-router/src/test/kotlin/azure/ApiKeysFunctionsTest.kt

```diff
@@ -792,8 +798,41 @@ class ApiKeysFunctionsTest {
             mockkObject(AuthenticatedClaims)
             every { AuthenticatedClaims.Companion.authenticate(any()) } returns claims
 
+            @Suppress("DEPRECATION")
             val response = ApiKeysFunctions().get(httpRequestMessage, "missing_org")
             assertThat(response.getStatus()).isEqualTo(HttpStatus.NOT_FOUND)
         }
+
+        @Test
+        fun `Test v1`() {
+            settings.organizationStore.put(
+                organization.name,
+                organization.makeCopyWithNewScopeAndJwk(defaultReportScope, jwk)
+                    .makeCopyWithNewScopeAndJwk(defaultReportScope, jwk2)
+                    .makeCopyWithNewScopeAndJwk(wildcardReportScope, jwk)
+            )
+
+            val httpRequestMessage = MockHttpRequestMessage()
+
+            val jwt = mapOf("organization" to listOf("DHSender_simple_reportAdmins"), "sub" to "test@cdc.gov")
+            val claims = AuthenticatedClaims(jwt, AuthenticationType.Okta)
+
+            mockkObject(AuthenticatedClaims)
+            every { AuthenticatedClaims.Companion.authenticate(any()) } returns claims
+
+            val response = ApiKeysFunctions().getV1(httpRequestMessage, organization.name)
+            assertThat(response.status).isEqualTo(HttpStatus.OK)
+            val jsonResponse = JSONObject(response.body.toString())
+            assertThat(jsonResponse.getJSONArray("data").length()).isEqualTo(2)
+            assertThat(jsonResponse.getJSONArray("data").map { obj -> (obj as JSONObject).getString("scope") })
+                .isEqualTo(
+                    listOf(defaultReportScope, wildcardReportScope)
+                )
+            assertThat(jsonResponse.getJSONArray("data").getJSONObject(0).getJSONArray("keys").length())
+                .isEqualTo(2)
+            val metaResponse = jsonResponse.getJSONObject("meta")
+            assertThat(metaResponse.getString("type")).isEqualTo("PublicKey")
+            assertThat(metaResponse.getInt("totalCount")).isEqualTo(2)
```

Comment:
Given the number of tests for the old function, it seems like maybe there should be more for the new one or the old ones should at least be adapted to work for the new function.

Reply:
So the two functions share 95% of the same code which is why I was relying on the old tests here, the only difference between the two endpoints code why is the branch [here](https://github.com/CDCgov/prime-reportstream/pull/9476/files#diff-996490aa05e4529bcc4be3a7d73d3b30c221dac71b2ff4ef9bf2083c0eb1b202R38).  Were there specific cases you were hoping to see covered?

Reply:
I was more trying to say that those tests are all marked as deprecated whereas I feel like they should be updated to test the new function. Does that make sense?

Reply:
Yea, so I was torn on this.  I'm not sure when the webex team will switch over, to the API under test now is the one that is actually in use in PROD.  The new API is just to get the ball rolling as well as demonstrating the wrapper class.

Reply:
I tried looking into adding https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-params, but it doesn't really save much duplication since there still needs to be branch to verify each different response shape.

I added the coverage result just for some clarity
<img width="826" alt="Screenshot 2023-05-15 at 9 50 31 PM" src="https://github.com/CDCgov/prime-reportstream/assets/1225895/55457607-7415-4d36-aa26-ff3ebf706784">

Reply:
Can we just duplicate the tests then? I know that is annoying but it would be nice not to lose those tests once the deprecation is gone.

Reply:
Added the tests, will definitely making cleaning up the code easier later